src/platform/guiqt.cpp

```diff
@@ -27,6 +27,9 @@
 #include <QSettings>
 #include <QTimer>
 
+#if WIN32
```

Comment:
I'd test the macro in a consistent manner (e.g. #ifdef/#ifndef) throughout the file.